src/.prettierrc

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
Can you please fix indentation on this one and add newline at the end? I suggest installing `final-newline` extension in VSCode to help out with that